Makefile

```diff
@@ -104,6 +100,17 @@ build-docker-api: .env
 	  $(COMPOSE) build --pull api
 .PHONY: build-docker-api
 
+build-docker-app: ## build the app container
+build-docker-app: \
+  .env \
+  build-docker-frontend \
+  build-frontend
+	WARREN_APP_IMAGE_BUILD_TARGET=$(WARREN_APP_IMAGE_BUILD_TARGET) \
+	WARREN_APP_IMAGE_NAME=$(WARREN_APP_IMAGE_NAME) \
+	WARREN_APP_IMAGE_TAG=$(WARREN_APP_IMAGE_TAG) \
+	  $(COMPOSE) build --pull app
```

Comment:
small question, why do we need a `--pull` option for the App and the API, and not for the frontend

Reply:
This is to ensure this plugin follows the latest `main` version of Warren. As the front-end image is not distributed, this is not required since we use an official Node.js base Docker image that is not supposed to change (except for security reasons).

Note that this is a temporary solution that needs to evolve to a pinned version of Warren once a (first?) release has been published.